components/selection/selection-mixin.js

```diff
@@ -87,6 +85,21 @@ export const SelectionMixin = superclass => class extends RtlMixin(superclass) {
 		return new SelectionInfo(keys, state);
 	}
 
+	setSelectionForAll(selected) {
```

Comment:
Related to @dbatiste's original comment here, still contemplating whether we want some kind of `setSelectionInfo` function.  It could take the selected value and then a key/node/whatever, and if no key is passed we could select/clear everything.  But we don't really have a use-case for that yet, so I'm leaning towards leaving it like this for now